Cargo.toml

```
@@ -13,13 +13,19 @@ Timeouts for futures.
 """
 
 [dependencies]
-
-[target.wasm32-unknown-unknown.dependencies]
-wasm-timer = "0.2.4"
-wasm-bindgen = "0.2.55"
-futures = "0.3.1"
-web-sys = "0.3.32"
+wasm-timer = { version = "0.2.4", optional = true }
+wasm-bindgen-crate = { package = "wasm-bindgen", version = "0.2.55", optional = true }
```

You don't have to rename the crate. `wasm-bindgen` can be the feature itself.

---

How do I make sure the other dependencies are enabled if I do that?

---

```suggestion
wasm-bindgen = { version = "0.2.55", optional = true }
```

Only this change :arrow_up: is needed. The content of your `[features]` section below will still work.

---

Are you sure? `Features and dependencies cannot have the same name: 'wasm-bindgen'`

---

I would have sworn this was working, so I did a bit of research and apparently they changed it: https://github.com/rust-lang/cargo/pull/5300